ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.1.yaml

```diff
@@ -79,49 +79,58 @@ releases:
       channel: fast
       version: "4.10"
   intermediate411:
-    release:
+    candidate:
       architecture: amd64
-      channel: fast
+      product: ocp
+      stream: nightly
       version: "4.11"
   intermediate412:
-    release:
+    candidate:
       architecture: amd64
-      channel: fast
+      product: ocp
+      stream: nightly
       version: "4.12"
   intermediate413:
-    release:
+    candidate:
       architecture: amd64
-      channel: fast
+      product: ocp
+      stream: nightly
       version: "4.13"
+  intermediate414:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
   latest:
     release:
       architecture: amd64
       channel: candidate
-      version: "4.12"
+      version: "4.13"
   target:
     candidate:
       architecture: amd64
       product: ocp
       stream: nightly
-      version: "4.14"
+      version: "4.15"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
 - as: aws-ipi-f30
-  cluster: build02
-  cron: 24 1 19 * *
+  cluster: build04
```

Comment:
For aws, perhaps we do not need to specify cluster ?

Reply:
I don't know how cluster works here. I remember @shellyyang1989 said there is performance issue https://redhat-internal.slack.com/archives/C02JEKXJ3JS/p1696987357273119?thread_ts=1696817499.718209&cid=C02JEKXJ3JS